src/views/ConfigurationJoystickView.vue

```diff
@@ -38,6 +38,17 @@
           <p>Could not stablish communication with the vehicle.</p>
           <p>Button functions will appear as numbers. If connection is restablished, function names will appear.</p>
         </div>
+        <div v-if="availableModifierKeys" class="flex items-center px-5 py-3 m-5 font-bold border rounded-md">
+          <Button
+            v-for="functionMapping in controllerStore.protocolMappings"
+            :key="functionMapping.name"
+            class="m-2"
+            :class="{ 'bg-slate-700': controllerStore.protocolMapping.name === functionMapping.name }"
+            @click="controllerStore.loadProtocolMapping(functionMapping)"
+          >
+            {{ functionMapping.name }}
+          </Button>
```

Comment:
I think the buttons would be more intuitive if they said something like "Restore default ROV controls" - general users don't need to know that we're calling them "functions mappings" internally, and the active "Restore" makes it clearer that the current mapping is going to be replaced by a different one, as opposed to the similarly styled modifier buttons directly below which just display different parts of the current controls mapping.

Tangentially related, having buttons for this that look the same as the buttons for the modified profiles (regular/shift) is unintuitive and somewhat unpleasant. That's probably out of scope for this PR though, since the preferred improvement is to change the modified profiles buttons into tabs (like in #617), and the new restore functions can be left as buttons.

Reply:
Agree with almost everything. The restore I'm not sure about as this is actually not a "restore" button, but a switch between mappings. I agree thought that it is currently *very* non-intuitive. I want to change this whole interface once we get the needed functionalities on (I think we are only missing the 32 buttons support after this PR).

Reply:
They are determined by the user when he changes it. They have default in the name because they start as default, but I agree with you on this. Will remove the "default" from the name.

Reply:
Done